.github/workflows/release_prep_hatch.yml

```diff
@@ -344,13 +350,13 @@ jobs:
       - name: "Remove Trailing Whitespace Via Pre-commit"
         continue-on-error: true
         run: |
-          pre-commit run trailing-whitespace --files dbt_common/__about__.py CHANGELOG.md .changes/*
+          pre-commit run trailing-whitespace --files dbt/adapters/__about__.py CHANGELOG.md .changes/*
           git status
       # this step will fail on newline errors but also correct them
       - name: "Removing Extra Newlines Via Pre-commit"
         continue-on-error: true
         run: |
-          pre-commit run end-of-file-fixer --files dbt_common/__about__.py CHANGELOG.md .changes/*
+          pre-commit run end-of-file-fixer --files dbt/adapters/__about__.py CHANGELOG.md .changes/*
```

Comment:
We should be able to modify these to stop failing.  Details in https://github.com/dbt-labs/dbt-release/issues/72

Reply:
This should be added to the `lint` `hatch` environment instead of here.